inputFiles/poromechanics/PoroDelftEggWellbore_base.xml

```diff
@@ -0,0 +1,232 @@
+<Problem>
+<!-- SPHINX_WELLBORE_TABLE -->
+<Functions>
+  <TableFunction 
+    name="timeFunction"
+    inputVarNames="{time}"
+    coordinates="{0.0, 0.1, 1e6}"
+    values="{0.0, 1.0, 1.0}" 
+  />
+</Functions>
+<!-- SPHINX_WELLBORE_TABLE_END -->
+
+<!-- SPHINX_WELLBORE_MATERIAL -->
+<Constitutive>
+  <PorousDelftEgg
+    name="porousRock"
+    solidModelName="rock"
+    porosityModelName="rockPorosity"
+    permeabilityModelName="rockPerm"
+  />
+
+  <DelftEgg
+    name="rock"
+    defaultDensity="2700"
+    defaultBulkModulus="10.0e9"
+    defaultShearModulus="6.0e9"     
+    defaultPreConsolidationPressure="-35.0e6"
+    defaultShapeParameter="6.5"
+    defaultCslSlope="1.2"
+    defaultVirginCompressionIndex="0.005"
+    defaultRecompressionIndex="0.001"
+  />
+
+  <CompressibleSinglePhaseFluid 
+    name="water"
+    defaultDensity="1000"
+    defaultViscosity="0.001"
+    referencePressure="0e6"
+    referenceDensity="1000"
+    compressibility="2.09028227021e-10"
```

Comment:
Are 11 digits of precision enough? 😃 Maybe we can stick to ~4.

Reply:
Right, no need for such precision